### DISCUSSION TITLE
Installation not working in ubuntu

### DISCUSSION THREAD

**Discussion by divyam7777** [aquasecurity/trivy#4906] [Bugs]
### Description

using https://aquasecurity.github.io/trivy/v0.44/getting-started/installation/ site for installing trivy in my pc but getting no package foubd trivy.

### Desired Behavior

installation should happen

### Actual Behavior

W: https://aquasecurity.github.io/trivy-repo/deb/dists/jammy/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package trivy
/home/vsts/work/_temp/3d968216-1c54-478a-a9ff-af2480dc5646.sh: line 6: trivy: command not found

### Reproduction Steps

```bash
1.
2.
3.
...
```

### Target

Virtual Machine Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Debug Output

```bash
installation failed
```

### Operating System

UBUNTU 22

### Version

```bash
0.44
```

### Checklist

- [ ] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
Duplicate of https://github.com/aquasecurity/trivy/issues/4908.
PS. We updated trivy-repo. Install should work correctly.
